Linkbrary/javascript/errorMessage.js

```diff
@@ -0,0 +1,40 @@
+// 에러시 적용되는 스타일
+// 에러메세지표시(id)
+function showidErrorMessage(message){ 
+  if(!document.getElementById('idErrorMessage')){
+    // input 색상변경
+    ID_INPUT.style.border = '3px solid #FF5B56';
+    // p 메세지표출
+    const errormessage = document.createElement("p");
+    errormessage.id = 'idErrorMessage'
+    errormessage.textContent = message;
+    errormessage.style.color = '#FF5B56';
+    IDBOX.appendChild(errormessage);
+  }
+}
+// 에러메세지표시(pw)
+function showpwErrorMessage(message){ 
+  if(!document.getElementById('pwErrorMessage')){
+    // input 색상변경
+    PW_TEXT[0].style.border = '3px solid #FF5B56';
+    // p 메세지표출
+    const errormessage = document.createElement("p");
+    errormessage.id = 'pwErrorMessage'
+    errormessage.textContent = message;
+    errormessage.style.color = '#FF5B56';
+    PWBOX[0].appendChild(errormessage);
+  }
+}
```

Comment:
위 두 함수는 거의 중복이라 inputDOM, id 만 추가 파라미터로 가져오면 공통화 할 수 있을 거 같아요!
```suggestion
function showidErrorMessage(message, inputDOM, id){ 
  if(!document.getElementById(id)){
    // input 색상변경
    inputDOM.style.border = '3px solid #FF5B56';
    // p 메세지표출
    const errormessage = document.createElement("p");
    errormessage.id = id
    errormessage.textContent = message;
    errormessage.style.color = '#FF5B56';
    IDBOX.appendChild(errormessage);
  }
}
```

Reply:
바로 아래 함수인 `showpwcheckErrorMessage` 도 다 공통화 가능할 듯 하네요!